docs/user-guide/v3_migration.rst

```diff
@@ -0,0 +1,226 @@
+3.0 Migration Guide
+===================
+
+Zarr-Python 3 represents a major refactor of the Zarr-Python codebase. Some of the
```

Comment:
```suggestion
Zarr-Python 3.0 represents a major refactor of the Zarr-Python codebase. Some of the
```
Or change 3.0 > 3 below - I'd have a soft preference for "3" over "3.0".

Reply:
changed to 3 in most places except where specifically referring to the 3.0 release.